flake8 on master is broken

If I run `./bin/flake8 --doctests src setup.py`
I get the following errors:
```
src/plone/recipe/zope2instance/recipe.py:299:37: F523 '...'.format(...) has unused arguments at position(s): 0
src/plone/recipe/zope2instance/recipe.py:721:36: E741 ambiguous variable name 'l'
```

--- a/news/139.bugfix
+++ b/news/139.bugfix
@@ -1,0 +1,1 @@
+Fix flake8 [ale-rt]

--- a/src/plone/recipe/zope2instance/recipe.py
+++ b/src/plone/recipe/zope2instance/recipe.py
@@ -296,7 +296,7 @@ class Recipe(Scripts):
             chameleon_cache = 'CHAMELEON_CACHE {}'.format(chameleon_cache)
             if environment_vars and '\n' in environment_vars:
                 # default case
-                environment_vars += '\n'.format(chameleon_cache)
+                environment_vars += '\n'
             elif environment_vars:
                 # handle case of all vars in one line
                 environment_vars += ' {}'.format(chameleon_cache)
@@ -717,9 +717,9 @@ class Recipe(Scripts):
             enable_products=enable_products,
             zope_conf_additional=zope_conf_additional,)
 
-        zope_conf = '\n'.join([l
-                               for l in zope_conf.splitlines()
-                               if l.rstrip()])
+        zope_conf = "\n".join(
+            [line for line in zope_conf.splitlines() if line.rstrip()]
+        )
         zope_conf_path = os.path.join(location, 'etc', 'zope.conf')
         with open(zope_conf_path, 'w') as f:
             f.write(zope_conf)

--- a/src/plone/recipe/zope2instance/recipe.py
+++ b/src/plone/recipe/zope2instance/recipe.py
@@ -296,7 +296,7 @@ class Recipe(Scripts):
             chameleon_cache = 'CHAMELEON_CACHE {}'.format(chameleon_cache)
             if environment_vars and '\n' in environment_vars:
                 # default case
-                environment_vars += '\n'
+                environment_vars += '\n{}'.format(chameleon_cache)
             elif environment_vars:
                 # handle case of all vars in one line
                 environment_vars += ' {}'.format(chameleon_cache)

--- a/src/plone/recipe/zope2instance/tests/zope2instance.txt
+++ b/src/plone/recipe/zope2instance/tests/zope2instance.txt
@@ -1445,8 +1445,11 @@ Our environment variables should be set now::
     >>> zope_conf = zope_conf.replace('\\', '/')
     >>> import re
     >>> env_vars = re.compile(r"<environment>\n\s*(?P<vars>.*)\n</environment>", re.M | re.S)
-    >>> re.search(env_vars, zope_conf).group('vars')
-    'TZ US/Eastern\nTMP /var/tmp\nDISABLE_PTS True'
+    >>> print(re.search(env_vars, zope_conf).group('vars'))
+    TZ US/Eastern
+    TMP /var/tmp
+    DISABLE_PTS True
+    CHAMELEON_CACHE .../var/cache
 
 Several all on one line::
 

--- a/src/plone/recipe/zope2instance/tests/zope2instance_zserver.txt
+++ b/src/plone/recipe/zope2instance/tests/zope2instance_zserver.txt
@@ -1737,8 +1737,10 @@ Our environment variables should be set now::
     >>> zope_conf = zope_conf.replace('\\', '/')
     >>> import re
     >>> env_vars = re.compile(r"<environment>\n\s*(?P<vars>.*)\n</environment>", re.M | re.S)
-    >>> re.search(env_vars, zope_conf).group('vars')
-    'TZ US/Eastern\nTMP /var/tmp\nDISABLE_PTS True'
+    >>> print(re.search(env_vars, zope_conf).group('vars'))
+    TZ US/Eastern
+    TMP /var/tmp
+    DISABLE_PTS True
 
 Several all on one line::
 
@@ -1770,8 +1772,10 @@ Our environment variables should be set now::
     >>> instance = os.path.join(sample_buildout, 'parts', 'instance')
     >>> zope_conf = open(os.path.join(instance, 'etc', 'zope.conf')).read()
     >>> zope_conf = zope_conf.replace('\\', '/')
-    >>> re.search(env_vars, zope_conf).group('vars')
-    'TZ US/Eastern\nTMP /var/tmp\nDISABLE_PTS True'
+    >>> print(re.search(env_vars, zope_conf).group('vars'))
+    TZ US/Eastern
+    TMP /var/tmp
+    DISABLE_PTS True
 
 
 HTTP server

